Cell.js

```diff
@@ -0,0 +1,19 @@
+class Cell {
+    constructor(
+        row,
+        col,
+        isMine = false,
+        mineNeighborCount = undefined,
```

Comment:
the naming is closer - sorry to beat a dead horse - but I think something like `neighboringMineCount` would be more concise